packages/logger/src/lib/global-log-level.ts

```diff
@@ -0,0 +1,11 @@
+import { LogLevel } from './model/log-level';
+
+let globalLogLevel = LogLevel.Debug;
```

Comment:
Of course you can. You just need to have access to the consumer.

Reply:
yes but thats the problem when you are trying to enable logs again with a bundled, minified app. You have no chance of doing that, since you don't know the mangled name, you cannot get the reference to anything.

Reply:
You mean for the dev? That is worth a separate package.

Reply:
I am thinking whether I should delete this file since, for most cases, the global logger instance is enough.

Reply:
Deleted this file.